transformations/negate_strengthen/transformation.py

```diff
@@ -0,0 +1,454 @@
+import random
+from interfaces.SentenceOperation import SentenceAndTargetOperation
+from tasks.TaskTypes import TaskType
+
+import re
+import spacy
+from spacy.tokenizer import Tokenizer
+import pandas as pd
+import nltk
+from nltk.corpus import wordnet
+from nltk.stem.wordnet import WordNetLemmatizer
+from nltk.metrics import edit_distance
+from nltk import pos_tag
+from pattern.en import conjugate as conjugate_en
+from typing import Union
+
+# defaults
+_RE_COMBINE_WHITESPACE = re.compile(r"\s+")
+MODAL_TYPE_LEMMA = ['be', 'to', 'have', 'get', 'do']
+MODAL_STRENGTHEN_DICT = {
+    'could': 'would',
+    'should': 'would',
+    'would': 'would',
+    'can': 'will',
+    'may': 'will',
+    'might': 'will',
+    'will': 'will'
+}
+
+# intialize
+try:
+    conjugate_en(verb='testing',tense='present',number='singular')
+except:
+    pass
+
+
+class NegateStrengthen(SentenceAndTargetOperation):
+    tasks = [TaskType.TEXT_CLASSIFICATION, TaskType.SENTIMENT_ANALYSIS]
+    languages = ["en"]
+    tgt_languages = ["en"]
+
+    def __init__(self, spacy_nlp=None, max_outputs=1, max_num_tries=2, seed=0, verbose=False):
```

Comment:
It's better not to make 'max_num_tries' configurable, instead to have a fixed max threshold value. 
As we are using this in recursion, if the user gave a very large value then recursion may result in an OutOfMemory issue.

Reply:
Ok then, I just changed the num_tries into a fixed value (at 2) within init

Reply:
Thanks, it looks fine now.